CHANGELOG.md

```diff
@@ -107,6 +109,107 @@ MIGraphX 2.8 for ROCm 6.0.0
 
 * Removed building Python 2.7 bindings
 
+#### HIP 6.0
+
+HIP 6.0 for ROCm 6.0.0
```

Comment:
Is this versioning correct? HIP 6.0 and ROCm 6.0.0? @samjwu

Reply:
Thanks for pointing that out. Fixed to HIP 6.0.0 in latest